templates/glean/defaults.schema.json

```diff
@@ -1,3 +1,7 @@
 {
-  "mozPipelineMetadata": false
+  "mozPipelineMetadata": {
+    "bq_dataset_family": "",
+    "bq_table": "",
+    "json_object_path": "object\\..*"
```

Comment:
I'm wondering if this shouldn't be more specific (`metrics\\.object\\..*`) in case someone names a metric `object`.

Reply:
Yeah, I think I need to adjust this in jsonschema-transpiler, which despite the internal naming, doesn't actually have the full path so we cannot match against that just yet.